assets/pages/bundles-landing/bundlesLandingReducers.js

```diff
@@ -14,6 +14,7 @@ export type ContribState = {
   type: Contrib,
   error: ?ContribError,
   amount: Amounts,
+  payPalError: string,
```

Comment:
I would normally do that yeah, but the `payPalContributionButton` takes a string for the error handler rather than a maybe string.

This is a large enough PR that I don't want to get into modifying that component as well, because it's on the critical path for payments on other parts of the site.